internal/kafka/kafka.go

```diff
@@ -94,8 +94,16 @@ func NewConsumerEventLoop(
 			case kafka.Error:
 				endpoints.IncConsumeErrors()
 				l.Log.Errorf("Consumer error: %v (%v)\n", e.Code(), e)
+			case kafka.OffsetsCommitted:
+				if e.Error != nil {
+					l.Log.Errorf("Unable to commit offset: %#v", e)
+					break
+				}
+
+				l.Log.Debugf("Ignored OffsetsComitted: %#v", e)
+
 			default:
-				l.Log.Infof("Ignored %v\n", e)
+				l.Log.Infof("Ignored %#v\n", e)
```

Comment:
Can you add a bit more context to this log message?  Maybe something like "Ignored kafka event"?  I was a bit stumped the first time I saw the original log message.  It took me a minute to figure out what I was seeing getting logged.

Reply:
Yes, that's a good idea too! Done!